### PR TITLE
[DC] Add S3 Get Bucket Tagging

### PR DESCRIPTION
**Pull request by edulop91**
ran it locally and made sure data populated in snowflake appropriately

### CODE REVIEW

File: src/connectors/aws_collect.py [hunk 1]
```diff
@@ -459,6 +459,14 @@
         ('target_grants', 'VARIANT'),
         ('target_prefix', 'STRING'),
     ],
+    # https://docs.aws.amazon.com/cli/latest/reference/s3api/get-bucket-tagging.html#output
+    's3_get_bucket_tagging': [
+        ('recorded_at', 'TIMESTAMP_LTZ'),
+        ('account_id', 'STRING'),
+        ('error', 'VARIANT'),
+        ('bucket', 'STRING'),
```
Comment (edulop91) [inline src/connectors/aws_collect.py:467]: I think I keep messing up the order of these

Comment (edulop91) [reply src/connectors/aws_collect.py:467]: although error then bucket seems to be the more common one

Comment (sfc-gh-afedorov) [reply src/connectors/aws_collect.py:467]: all good, it's an unfortunate limitation. the params go before error, and then either "error" or everything after it is populated with response. if all of them are NULL then there was no error and the response was empty.

Comment (sfc-gh-afedorov) [reply src/connectors/aws_collect.py:467]: bucket should be after error if the bucket is part of the response and before the error if it's part of the query

Comment (edulop91) [reply src/connectors/aws_collect.py:467]: thank you

**Review by sfc-gh-afedorov — APPROVED** [review]

**Pull request merged**